proposals/low-level-struct-improvements.md

```diff
@@ -939,17 +969,133 @@ This will work but is likely going to result in unnecessary code generation. One
 1. `Unsafe.AsRef<T>(in T value)` could expand its existing purpose by changing to `scoped in T value`. This would allow it to both remove `in` and `scoped` from parameters. It then becomes the universal "remove ref safety" method
 2. Introduce a new method whose entire purpose is to remove `scoped`: `ref T Unsafe.AsUnscoped<T>(scoped in T value)`. This removes `in` as well because if it did not then callers still need a combination of method calls to "remove ref safety" at which point the existing solution is likely sufficient.
 
-### What will make C# 11.0
+### Unscoped this by default?
+The design only has two locations which are `scoped` by default: 
+
+- `this` is `scoped ref` 
+- `out` is `ref scoped`
+
+The decision on `out` is to significantly reduce the compat burden of `ref` fields and at the same time is a more natural default. It lets developers actually think of `out` as data flowing outward only where as if it's `scoped ref` or even `ref` then the rules must consider data flowing in both directions. This leads to significant developer confusion where as the `ref scoped` interpretation means that `out` means `out`. 
+
+The decision on `this` this though is undesirable because it means a `struct` cannot return a field by `ref`. This is an important scenario to high perf developers and the `unscoped` keyword was added effectively for this one scenario.
```

Comment:
>was added effectively [](http://example.com/codeflow?start=191&length=21)

would help

Reply:
Tried rewording this a bit. Let me know if it reads better now.